backend/cybnetics/endpoints.py

```diff
@@ -1,118 +1,58 @@
-import json
-import ast
 import jwt
-import datetime
+from flask import Flask, jsonify, request
+from pymongo import MongoClient
 
-from flask import Flask, jsonify, request, session
 
 app = Flask(__name__)
 app.config.from_pyfile('./config.py')
 
 
-db = app.config['DATABASE']
+@app.before_first_request
+def db_connect():
+    app.db = MongoClient(app.config['DB_URI']).get_default_database()
```

Comment:
in resources/db we can just have db=None and the here we can set db=app.db

Reply:
Not sure exactly what you are saying here so I will just leave this for you.